### PR TITLE
reposition left menu (docs ToC) & make sure it doesn't overlap with other components

### LINKED ISSUE #2352: Left navbar doesn't scroll properly on the docs site
# Bug Report 

**What happened**:
The navbar on the left on the docs site doesn't scroll properly, it seems fixed at one point.

**What you expected to happen**:
The navbar should scroll along with the rest of the page.

**How to reproduce it (as minimally and precisely as possible)**:
Scroll to the bottom of https://svc-cat.io/docs/resources/

**Anything else we need to know?**:
image

(edit from Carolyn): Here's the docs on how to work with our documentation site: https://svc-cat.io/docs/devguide/#documentation

### PR DESCRIPTION
**Pull request by arief-hidayat**
This PR is a 
 - [ ] Feature Implementation
 - [x] Bug Fix
 - [ ] Documentation

**What this PR does / why we need it**:
* make sure the left menu doesn't jump out of frame (overlap with the components such as footer)
* improve the positioning (recalculate the top instead of just fixing the position)

**Which issue(s) this PR fixes** 

Fixes #2352

Please leave this checklist in the PR comment so that maintainers can ensure a good PR.

Merge Checklist:
 - [ ] New feature 
   - [ ] Tests
   - [ ] Documentation
 - [ ] SVCat CLI flag
 - [ ] Server Flag for config
   - [ ] Chart changes
   - [ ] removing a flag by marking deprecated and hiding to avoid
         breaking the chart release and existing clients who provide a
         flag that will get an error when they try to update

### CODE REVIEW

**Comment by arief-hidayat** [comment]
Here is the screen recording: https://youtu.be/NGUkEkFQKW8

File: docsite/js/script.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
  //modal close button
 (function(){
     //π.modalCloseButton = function(closingFunction){
-    //	return π.button('pi-modal-close-button', null, null, closingFunction);
+    // return π.button('pi-modal-close-button', null, null, closingFunction);
     //};
```
Comment (MHBauer) [inline docsite/js/script.js]: This is all commented out? Can we drop it?

**Comment by MHBauer** [comment]
/ok-to-test
doesn't actually need to be tested, and the netflify output looks good.

/lgtm
one nit

File: docsite/js/script.js (hunk 1 above)
Comment (arief-hidayat) [reply docsite/js/script.js]: It's actually not related to my change.
But I think it's better to remove those commented lines.

Comment (arief-hidayat) [reply docsite/js/script.js]: I have removed it. thanks

**Comment by MHBauer** [comment]
maybe we needed that, now it looks funny

screen shot 2019-03-05 at 17 42 32

**Comment by arief-hidayat** [comment]
My bad. I accidentally included other change in the commit.
```
'overflow' : 'scroll'
```
I'll raise new PR to revert it.

**Pull request merged**